### PR TITLE
uasc-44/admin-dashboard-layout added RequestBlock component

### PR DESCRIPTION
**Pull request by svimi**
ive created a request block component to render any data we want to fetch from firebase. at the moment it only works with dummy data manually put in. potential changes in the future can include scroll bar stylization for the request list, among other things.

### CODE REVIEW

File: frontend/src/components/AdminRequests.js [hunk 1]
```diff
@@ -1,220 +1,90 @@
...
-                      DATE CHANGE
-                      <span style={textBreakStyle}></span>
-                      5/7 - 7/7 -&gt; 7/7 - 9/7
-                    </Typography>
-                  </Box>
-                </Paper>
-              </Grid>
+              {requests.map((request, index) => (
+                <RequestBlock
+                  key={request.user_id}
```
Comment (OverHash) [inline frontend/src/components/AdminRequests.js:83]: A user ID isn't a unique key (a user can have multiple requests). We should use the request document ID returned by Firebase, something like
```js
key={request._id}
```

File: frontend/src/components/RequestDetails.js
Comment (BW82262) [inline frontend/src/components/RequestDetails.js:1]: I'm assuming this is used for your next user story?

**Review by BW82262 — APPROVED** [review]
lgtm

File: frontend/src/components/AdminRequests.js [hunk 2]
```diff
@@ -22,8 +22,8 @@ const AdminDashboardRequest = () => {
         {
           user_id: "whatsgood",
           booking_id: "8mYj7rWOMH6hGy4FzMed",
-          query: "I'd like to cancel my booking.",
-          query_type: "cancellation",
+          query: "I'd like to change my booking.",
+          query_type: "date change",
```
Comment (OverHash) [inline frontend/src/components/AdminRequests.js:26]:
```suggestion
          query_type: "dateChange",
```

**Review by OverHash — CHANGES_REQUESTED** [review]

File: frontend/src/components/AdminRequests.js (hunk 2 above)
Comment (svimi) [reply frontend/src/components/AdminRequests.js]: changed

File: frontend/src/components/RequestDetails.js
Comment (svimi) [reply frontend/src/components/RequestDetails.js:1]: yea

File: frontend/src/components/AdminRequests.js (hunk 1 above)
Comment (svimi) [reply frontend/src/components/AdminRequests.js]: is it okay to just use the booking id? thats what ive set it as right now

**Review by OverHash — CHANGES_REQUESTED** [review]
Please delete `node_modules` folder in root and the root `package.json`.

**Pull request merged**